source/common/http/utility.cc

```diff
@@ -73,17 +73,12 @@ void Utility::appendXff(HeaderMap& headers, const Network::Address::Instance& re
     return;
   }
 
-  HeaderString& header = headers.insertForwardedFor().value();
   const std::string& address_as_string = remote_address.ip()->addressAsString();
-  HeaderMapImpl::appendToHeader(header, address_as_string.c_str());
+  headers.appendForwardedFor(address_as_string.c_str(), ",");
```

Comment:
s/.c_str()//

Your append methods take a string_view as an arg. It's faster to construct a string_view from a string than a char* as the char* version must call strlen.

Reply:
Done.